scripts/sed_links.sh

```diff
@@ -24,6 +24,25 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
     sed -i '' 's|(/cloud/security/secure-s3#access-your-s3-bucket-with-the-clickhouseaccess-role)|(/cloud/data-sources/secure-s3#access-your-s3-bucket-with-the-clickhouseaccess-role)|g' docs/sql-reference/table-functions/s3.md
     sed -i '' 's|(/cloud/security/secure-s3#access-your-s3-bucket-with-the-clickhouseaccess-role)|(/cloud/data-sources/secure-s3#access-your-s3-bucket-with-the-clickhouseaccess-role)|g' docs/sql-reference/table-functions/s3Cluster.md
     sed -i '' 's|(#cuttofirstsignificantsubdomaincustom)|(#cutToFirstSignificantSubdomainCustom)|g' docs/sql-reference/functions/url-functions.md
+    # Fix broken links in interface docs
```

Comment:
This script gets run at build time when the ClickHouse/ClickHouse docs get pulled into this repo to build the docs. There's unfortunately no way to selectively exclude files from the built-in broken link checker of Docusaurus so we need to trick it here. I will open a PR on ClickHouse/ClickHouse next to remove the "old" files and then we can remove these.